### DISCUSSION TITLE
If I build DS on the new machine and want to access the existing Kerberos enabled environment, what should I do？

### DISCUSSION THREAD

**Discussion by cw1322311203** [apache/dolphinscheduler#11712] [Q&A] [answered]
If I build DS on the new machine and want to access the existing Kerberos enabled environment, what should I do？Do I need to install Kerberos on the DS machine？

**Comment by SbloodyS** [comment]
What components of DS do you want to use?

**Reply by cw1322311203** [reply]
flink,shell,sql

**Reply by cw1322311203** [reply]
version is DS 2.0.5

**Comment by DarkAssassinator** [comment, marked as answer]
i think so, may u should install the Kerberos in the DS worker machine first, then use the shell task.

**Comment by wsczm** [comment]
你好 请问这个问题解决了吗 我的也是2.0.5 ds上配置的kerberos的认证 没有生效
